ui/src/embeddable/reducers/data-api.js

```diff
@@ -74,6 +77,7 @@ export const getData = ({ source, app, params }) => {
     return get(APIS[app] + countryId);
   } else if (app === NUMBER_OF_VARIETIES_RELEASED
     || app === AVAILABILITY_OF_BASIC_SEED
+    || app === MARKET_CONCENTRATION_HHI  
```

Comment:
for further charts take a look how numberActiveCompanies was configured to avoid adding api info to the ui and leave it to config